### ISSUE TITLE
Possible to record into a fixed length buffer? (i.e. keep the last X seconds)

### ISSUE THREAD

**Issue by v2k** [RenderHeads/UnityPlugin-AVProMovieCapture#85]
The ability to record to a buffer, keeping the last X seconds of video:

Constantly recording and keeping the last 10-15 seconds 

- An option to keep the last X seconds of recording.

A possibly alternative is to use the output as an Image Sequence and constantly delete old images.

**Comment by kahnivore** [comment]
Yeah, it'd be great to support this, however it is tricky to do because we're creating mp4 files which are baked in once they're written to disk. We currently don't have plans to add this feature due to the amount of work it would be. 

Your workaround suggestion is good and you could implement this, it's just not something we would implement because it's too specific of a use case for us to add as a feature.
